code/script.js

```diff
@@ -26,22 +34,130 @@ const showMessage = (message, sender) => {
       </section>
     `
   }
+
   // This little thing makes the chat scroll to the last message when there are too many to be shown in the chat box
   chat.scrollTop = chat.scrollHeight
 }
 
 // Starts here
 const greeting = () => {
-  showMessage(`Hello there, What's your name?`, 'bot')
-  // Just to check it out, change 'bot' to 'user' here 👆
+  showMessage(`Welcome to the Travelbot! What's your name?`, 'bot')
+}
+
+const continentQuestion = () => {
+  showMessage(`Hello ${yourName}! Which continent would you like to visit?`, 'bot')
+  inputWrapper.innerHTML = /*html*/`
+    <select id="select">
+      <option disabled selected value="">Pick a continent</option>
+      <option id="africa" value="Africa">Africa</option>
+      <option id="europe" value="Europe">Europe</option>
+      <option id="south-america" value="South America">South America</option>
+      <option id="north-america" value="North America">North America</option>
+      <option id="asia" value="Asia">Asia</option>
+      <option id="antarctica" value="Antarctica">Antarctica</option>
+      <option id="oceania" value="Oceania">Oceania</option>
+   </select>
+  `
+  const select = document.getElementById('select')
+  select.addEventListener('change', () => handleContinentQuestion(select.value))
+}
+
+const weatherQuestion = () => {
+  showMessage('What kind of weather do you prefer?', 'bot')
+  inputWrapper.innerHTML = /*html*/ `
+  <button id="sun" value="sun">Sunny</button>
+  <button id="cloud" value="cloud">Cloudy</button>
+  <button id="snow" value="snow">Snowy</button>
+  `
+  document
+  .getElementById('sun')
+  .addEventListener('click', () => handleWeatherQuestion('sunny'))
```

Comment:
if you wanted to, you could add another global variable for 'weather' const weather = "" and at this point (inside each separate event listener) you could change the value of the weather global variable. Then you could access that global later inside your 'book' function, exactly like you did for 'continent', so that your message was even more specific.